RevenueCatUI/Helpers/ColorInformation+MultiScheme.swift

```diff
@@ -21,6 +21,15 @@ extension PaywallData.Configuration.ColorInformation {
             return light
         }
 
+        return .comine(light: light, dark: dark)
+    }
+
+}
+
+extension PaywallData.Configuration.Colors {
+
+    @available(iOS 14.0, macOS 11.0, tvOS 14.0, watchOS 7.0, *)
+    static func comine(light: Self, dark: Self) -> Self {
```

Comment:
what's `comine`? I've never heard the term before. Is this meant to be `combine`?

Reply:
It's a common term in some secret functional programming circles......

But yes 💩

Reply:
I have also never heard of it and I know feel better that @aboedo also doesn't know 😛